src/printer.ts

```diff
@@ -773,6 +775,20 @@ export class PugPrinter {
 			if (token.val !== true) {
 				this.result += `=${token.val}`;
 			}
+		} else if (token.name === 'class' && this.options.pugClassNotation === 'attribute') {
+			const val: string = token.val.slice(1, -1).trim();
```

Comment:
**thought:** Havn't tested the code yet, and I'm currently just in GitHub Web UI as writing this review.
But this looks dangerous, cause the `token.val` could be not quoted and include js code 🤔, could it?

Reply:
You should add a test case for this 🙂